bigquery/datasets.js

```diff
@@ -15,123 +15,117 @@
 
 'use strict';
 
-const BigQuery = require('@google-cloud/bigquery');
+function createDataset (datasetId, projectId) {
+  // [START bigquery_create_dataset]
+  // Imports the Google Cloud client library
+  const BigQuery = require('@google-cloud/bigquery');
+
+  // The project ID to use, e.g. "your-project-id"
+  // const projectId = "your-project-id";
```

Comment:
If we do specify them, we should also test to make sure they are (properly) configurable - i.e. test the sample without `GCLOUD_PROJECT` being set (or being set to the empty string).